apps/sms/js/link_helper.js

```diff
@@ -58,7 +58,7 @@ function checkDomain(domain) {
 }
 
 // defines things that can match right before to be a "safe" link
-var safeStart = /[\s,:;\(>]/;
+var safeStart = /[\s,:;\(>\u0080-\uFFFF]/;
```

Comment:
Well... full-width char might be used in Chinese/Japan/Korea IME, and it's always painful for developers ;) For this case you might be likely to see a phone number will all full-width symbol and number like `（６６６）６６６６`, and our logic will even not treat this string as valid phone number